panic: runtime error: index out of range

```
$ swm --help
panic: runtime error: index out of range

goroutine 1 [running]:
main.getDefaultStory(0x785d40, 0xc420020480)
        /home/kalbasit/.filesystem/src/github.com/kalbasit/swm/cmd/swm/main.go:147 +0x1c3
main.main()
        /home/kalbasit/.filesystem/src/github.com/kalbasit/swm/cmd/swm/main.go:25 +0xdb
$ swm --version
panic: runtime error: index out of range

goroutine 1 [running]:
main.getDefaultStory(0x785d40, 0xc420020480)
        /home/kalbasit/.filesystem/src/github.com/kalbasit/swm/cmd/swm/main.go:147 +0x1c3
main.main()
        /home/kalbasit/.filesystem/src/github.com/kalbasit/swm/cmd/swm/main.go:25 +0xdb
 $ echo $TMUX
/tmp/tmux-1026/default,2014,0
```

--- a/cmd/swm/main.go
+++ b/cmd/swm/main.go
@@ -125,7 +125,10 @@ func getDefaultProfile() string {
 	tmuxSocketPath := os.Getenv("TMUX")
 	if tmuxSocketPath != "" {
 		profileStory := strings.Split(path.Base(tmuxSocketPath), ",")[0]
-		p = strings.Split(profileStory, "@")[0]
+		profileStoryArr := strings.Split(profileStory, "@")
+		if len(profileStoryArr) == 2 {
+			p = profileStoryArr[0]
+		}
 	}
 
 	if p == "" {
@@ -144,7 +147,10 @@ func getDefaultStory() string {
 	tmuxSocketPath := os.Getenv("TMUX")
 	if tmuxSocketPath != "" {
 		profileStory := strings.Split(path.Base(tmuxSocketPath), ",")[0]
-		s = strings.Split(profileStory, "@")[1]
+		profileStoryArr := strings.Split(profileStory, "@")
+		if len(profileStoryArr) == 2 {
+			s = profileStoryArr[1]
+		}
 	}
 	if s == "" {
 		i3Workspace, err := getActiveI3WorkspaceName()

